apps/web/src/lang/fr.json

```
@@ -7558,6 +7594,10 @@
     "defaultMessage": "Que puis-je faire si j'ai supprimé l'application ou changé de téléphone et que je n'ai plus les codes de récupération?",
     "description": "First of the Frequently Asked Questions for logging in"
   },
+  "m+M6WO": {
+    "defaultMessage": "L'équipe des talents numériques du GC examinera et publiera votre annonce.",
```

If the English is **The GC Digital Talent team**, then this needs to be changed accordingly.
```suggestion
    "defaultMessage": "L'équipe des Talents numériques du GC examinera et publiera votre annonce.",
```